devenv.nix

```diff
@@ -255,7 +255,13 @@ in {
       type = lib.types.listOf lib.types.package;
       default = [ ];
       example = [ pkgs.jpegoptim pkgs.optipng pkgs.gifsicle ];
-      description = "Additional packages to be installed";
+      description = ''Additional packages to be installed'';
+    };
+
+    enableMysqlBinLog = lib.mkOption {
+      type = lib.types.bool;
+      default = true;
```

Comment:
let's deactivate the binlog by default, i don't see a huge benefit of enabling it but its an option if needed.

Reply:
Okay, I did not want to introduce a change of default behaviour, i.e. users wondering where their bin logs went.

Reply:
This has been changed.